src/events/cd-event-list.vue

```diff
@@ -32,14 +32,21 @@
       </div>
       <div class="cd-event-list__event-view-wrapper">
         <router-link :to="{name: 'EventDobVerification', params: {eventId: event.id}}"
+                     v-if="!isEventFull(event.id)"
                      tag="button" class="btn btn-lg btn-primary cd-event-list__event-view">
           {{ $t('See Details and Book') }}
         </router-link>
+        <router-link :to="{name: 'EventDobVerification', params: {eventId: event.id}}"
+                     v-if="isEventFull(event.id)" disabled="disabled"
+                     tag="button" class="btn btn-lg cd-event-list__event-view cd-event-list__event-view--disabled">
+          {{ $t('FULL') }}
```

Comment:
Should use `Full` instead, and use `text-transform: uppercase;` to make it all caps instead. Makes it more reusable for other situations where we don't want it all caps.